packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -420,7 +420,7 @@ function handleUploadDataChunkError({
    * }
    */
   const systemErrorResponseData = getSystemErrorResponseData(err);
-
+  delete err?.config?.data;
```

Comment:
Should this data continued to be logged but at a different level? Or is it never helpful for debugging?